service/src/main/kotlin/fi/espoo/evaka/jamix/JamixService.kt

```diff
@@ -77,7 +80,19 @@ class JamixService(
         }
     }
 
-    fun syncDiets(db: Database.Connection, clock: EvakaClock) {
+    fun planDietSync(db: Database.Connection, clock: EvakaClock) {
+        db.transaction { tx ->
+            tx.removeUnclaimedJobs(setOf(AsyncJobType(AsyncJob.SyncJamixDiets::class)))
```

Comment:
Pitäisikö tästä lokittaa jos joudutaan poistamaan taskeja?
```suggestion
            val count = tx.removeUnclaimedJobs(setOf(AsyncJobType(AsyncJob.SyncJamixDiets::class)))
            logger.warn("Removed $count unclaimed async jobs")
```